### ISSUE TITLE
Tell us about this site form

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#75]
A "Tell us about this site" form to be submitted to ORG will let the user state whether site is adult, re-direct, squat, for sale, dead etc.

This information can be displayed on the links within the browser, eg "This website appears to be sqautted and is likely to be blocked legitimately."

Once a report has been submittedto ORG, direct to thank you for reporting, click here to check another site - link to random site.

**Comment by JimKillock** [comment]
I had a look at the form. Could we make some changes to the content, eg:

 <i>This domain is being squatted, but contains adult material</i>

I think this should be without qualification - ie just "This domain is being squatted".

I think the form should be multi option so this can be selected alongside "is adult"

**Comment by dantheta** [comment]
Those changes are done now.

**Comment by JimKillock** [comment]
Thanks @dantheta. How about also some positive responses, eg:

"This site appears to be free from adult content" just in case that is helpful to us?

**Comment by pamcowburn** [comment]
Where can I see the form?

**Comment by dantheta** [comment]
Find a blocked site through the browser and click on the flag icon on that row.

On 1 November 2016 14:17:48 GMT+00:00, pamcowburn notifications@github.com wrote:

> Where can I see the form?
> 
> ## 
> 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/openrightsgroup/blocked-org-uk/issues/75#issuecomment-257577568

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by pamcowburn** [comment]
Ok thanks

**Comment by dantheta** [comment]
> Thanks @dantheta. How about also some positive responses, eg:
> 
> "This site appears to be free from adult content" just in case that is helpful to us?

That's done now.

**Comment by pamcowburn** [comment]
Re:  "This site appears to be free from adult content" - do we think people could find it difficult to categorise in some cases e.g. some sites are clearly intended for adults but their content is not necessarily harmful for kids.

**Comment by JimKillock** [comment]
@pamcowburn Yes, sometimes it will be hard to know. People also don't know what categories are blocked by each ISP. We could include a link to a checklist or similar?

**Comment by pamcowburn** [comment]
I wondered if we could have a - this site is aimed towards adults but does not pose harm to kids

**Comment by pamcowburn** [comment]
Sorry posted too soon. We should test with people and see what they come up with.

**Comment by dantheta** [comment]
Yep - this additional categorisation doesn't have to be spot-on to start with.  It is for our use, rather than something that the ISPs will be exposed to.

**Comment by dantheta** [comment]
Any other changes for this one? I'm spinning off the frontend advisory functionality (the "this site has been squatted") to a separate ticket.
